src/Themes.scss

```diff
@@ -1415,8 +1415,8 @@ body.contrast-theme {
         }
     }
 
-    button.btn.ModalFooter__cancelButton,
-    button.btn.ModalFooter__doneButton {
+    button.btn.ModalWithFooter__secondaryButton,
+    button.btn.ModalWithFooter__primaryButton {
```

Comment:
Until we remove the usage of ModalFooter (still used in the keybaord dialog), we'll need both sets of styles.